OpenRA.Game/Server/ServerOrder.cs

```diff
@@ -55,9 +59,23 @@ public byte[] Serialize()
 			var ms = new MemoryStream(1 + Name.Length + 1 + Data.Length + 1);
```

Comment:
Add an extra 5 bytes to the initial capacity to account for the fields byte and possibility for ExtraData.